server/src/main/java/org/elasticsearch/action/admin/indices/resolve/ResolveIndexAction.java

```diff
@@ -439,7 +440,7 @@ public static class TransportAction extends HandledTransportAction<Request, Resp
         private final ThreadPool threadPool;
         private final ClusterService clusterService;
         private final RemoteClusterService remoteClusterService;
-        private final IndexAbstractionResolver indexAbstractionResolver;
+        private final IndexNameExpressionResolver indexAbstractionResolver;
```

Comment:
Nit: can we rename the variable accordingly as well?

Reply:
pushed [9c336dd](https://github.com/elastic/elasticsearch/pull/92820/commits/9c336dd6f14844a928949841cb2dcb65d853df46)